app/src/main/java/com/framgia/soundclound/data/source/local/sqlite/TrackLocalDataSource.java

```diff
@@ -0,0 +1,55 @@
+package com.framgia.soundclound.data.source.local.sqlite;
+
+import android.content.Context;
+
+import com.framgia.soundclound.data.model.Album;
+import com.framgia.soundclound.data.model.Track;
+import com.framgia.soundclound.data.source.TrackDataSource;
+import com.framgia.soundclound.util.Constant;
+
+import java.util.List;
+
+/**
+ * Created by Bui Danh Nam on 10/1/2018.
+ */
+
+public class TrackLocalDataSource implements TrackDataSource.LocalDataSource {
+    private static TrackLocalDataSource INSTANCE;
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName